src/reducers.ts

```diff
@@ -1,20 +1,18 @@
 import * as vortexApi from "vortex-api"; // eslint-disable-line import/no-extraneous-dependencies
 import * as actions from './actions';
+import {
+  DynamicFeature,
+  DynamicFeatureDefaults,
+} from "./features";
+import { VORTEX_STORE_PATHS } from "./index.metadata";
+import { nestedRecordFrom } from "./util.functions";
+import { VortexReducerSpec } from "./vortex-wrapper";
 
-/**
- * reducer for changes to ephemeral session state
- */
-const settingsReducer: vortexApi.types.IReducerSpec = {
+
+export const makeSettingsReducer = (settingsDefaults: DynamicFeatureDefaults): VortexReducerSpec => ({
   reducers: {
-    [actions.setArchiveAutoConvert as any]: (state, payload) =>
-      vortexApi.util.setSafe(state, [`v2077_feature_redmod_autoconvert_archives`], payload),
-    [actions.setRedmodForceDeploy as any]: (state, payload) =>
-      vortexApi.util.setSafe(state, [`redmodForceDeploy`, payload.profileId], payload.force),
-  },
-  defaults: {
-    archiveAutoConvert: false,
-    redmodForceDeploy: {},
+    [actions.setREDmodAutoconvertArchivesAction.toString()]: (state, payload: boolean) =>
```

Comment:
oh, you found it is a string. I didn't know where to look to find that info. :/

Reply:
Yeah, ran into it somewhere in the Redux docs